sdk/identity/azure-identity/src/main/java/com/azure/identity/WorkloadIdentityCredentialBuilder.java

```diff
@@ -66,6 +67,19 @@ public WorkloadIdentityCredentialBuilder tokenFilePath(String tokenFilePath) {
         return this;
     }
 
+    /**
+     * Enables the Kubernetes token proxy feature for AKS workload identity scenarios.
+     * When enabled, the credential will attempt to use a custom token proxy configured through
+     * environment variables (AZURE_KUBERNETES_TOKEN_PROXY, AZURE_KUBERNETES_CA_FILE,
+     * AZURE_KUBERNETES_CA_DATA, AZURE_KUBERNETES_SNI_NAME).
+     *
+     * @return An updated instance of this builder with Kubernetes token proxy enabled.
+     */
+    public WorkloadIdentityCredentialBuilder enableKubernetesTokenProxy() {
```

Comment:
```suggestion
    public WorkloadIdentityCredentialBuilder enableAzureTokenProxy() {
```
I'm open to other suggestions; my thinking here is:
1. it's a token proxy
2. it requires Azure
3. it does not require Azure Kubernetes Service
4. "Kubernetes" is implied because this credential type is explicitly for k8s apps